src/utils/actions.ts

```diff
@@ -0,0 +1,36 @@
+import { FluxStandardAction, isFSA } from "flux-standard-action";
+
+// Certain actions that we should never replay across stores
+const blacklist = [/^@@/, /^redux-form/];
```

Comment:
also seems something that we might consider configurable in the future (but maybe not now)